python/ray/util/spark/databricks_hook.py

```diff
@@ -9,25 +9,15 @@
 _logger = logging.getLogger(__name__)
 
 
-class _NoDbutilsError(Exception):
-    pass
-
-
-def get_dbutils():
+def get_db_entry_point():
     """
-    Get databricks runtime dbutils module.
+    Return databricks entry_point instance, it is for calling some
+    internal API in databricks runtime
     """
-    try:
-        import IPython
+    from dbruntime import UserNamespaceInitializer
 
-        ip_shell = IPython.get_ipython()
-        if ip_shell is None:
-            raise _NoDbutilsError
-        return ip_shell.ns_table["user_global"]["dbutils"]
-    except ImportError:
-        raise _NoDbutilsError
-    except KeyError:
-        raise _NoDbutilsError
+    user_namespace_initializer = UserNamespaceInitializer.getOrCreate()
+    return user_namespace_initializer.get_spark_entry_point()
```

Comment:
Note:

I synced with databricks colleagues, we shouldn't use `dbutils.entry_point` or `dbutils.jobs.entry_point` to get entry_point, but we should use UserNamespaceInitializer instead.